3 - PB/Documentazione esterna/Analisi dei Requisiti/requirements.json

```diff
@@ -0,0 +1,1293 @@
+			"cat": "Obbligatorio",
+			"achieved": false
+		},
+		{
+			"id": "FM-154",
+			"source": "UC–25.3.2",
+			"description": "L'utente deve visualizzare un errore di connessione se l'accesso all'API non è possibile",
+			"cat": "Obbligatorio",
+			"achieved": false
+		},
+		{
+			"id": "FD-155",
+			"source": "UC–25.3",
+			"description": "Quando un ordine di movimentazione è convalidato, esso viene inserito in una cronologia delle operazioni accessibile dall'utente",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FM-156",
+			"source": "UC–25.3",
+			"description": "Quando un ordine di movimentazione è rifiutato, il prodotto oggetto dell'operazione ritorna nella posizione di partenza",
+			"cat": "Obbligatorio",
+			"achieved": false
+		},
+		{
+			"id": "FD-157",
+			"source": "UC–26",
+			"description": "L'utente deve poter visualizzare la cronologia degli ordini di movimentazione convalidati",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FD-158",
+			"source": "UC–26",
+			"description": "L'utente deve poter visualizzare l'ID del bin di partenza degli ordini di movimentazione convalidati se l'operazione è partita da un bin",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FD-159",
+			"source": "UC–26.1.2",
+			"description": "L'utente deve poter visualizzare l'ID del bin di destinazione dell'ordine di movimentazione convalidato",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FD-160",
+			"source": "UC–26.1.1",
+			"description": "L'utente deve poter visualizzare l'ID del prodotto oggetto dell'ordine di movimentazione convalidato",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FD-161",
+			"source": "UC–26.1.1",
+			"description": "L'utente deve poter visualizzare il nome del prodotto oggetto dell'ordine di movimentazione convalidato",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FD-162",
+			"source": "UC–26.1.1",
+			"description": "L'utente deve poter visualizzare l'ID del bin di partenza degli ordini di movimentazione convalidati se l'operazione è partita da un bin",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FD-163",
+			"source": "UC–27",
+			"description": "L'utente può poter ispezionare un singolo ordine di movimentazione convalidato",
+			"cat": "Desiderabile",
+			"achieved": false
+		},
+		{
+			"id": "FM-164",
+			"source": "UC–27.1",
+			"description": "Il sistema deve evidenziare graficamente il bin di destinazione dell'ordine di movimentazione ispezionato",
+			"cat": "Obbligatorio",
+			"achieved": false
+		},
+		{
+			"id": "FM-165",
+			"source": "UC–27.2",
+			"description": "Se l'ordine di movimentazione ispezionato si è originato da un bin, il sistema deve evidenziare graficamente il bin di partenza nell'ambiente 3D",
+			"cat": "Obbligatorio",
+			"achieved": false
+		}
+	],
+	"quality": [
```

Comment:
Nei requisiti di qualità è stato invertito l'ordine di presentazione di `source` e `cat`, nella tabella vengono visualizzati in modo errato